investigator_message_metrics/overlays/cnv-prod/route.yaml

```diff
@@ -0,0 +1,9 @@
+---
```

Comment:
the route and the service can move to the base, there is no cluster-based changes here.